File: Application/Services/Operations/StockProduct/Dtos/ProductKind/ModelDto.cs


using System.Collections.Generic;
using Application.Services.Shared.Dtos;

namespace Application.Services.Operations.StockProduct.ProductKind
{
    public class ModelDto : RootBaseDto
    {
        public string Name { get; set; }
        public int ManufacturerId { get; set; }
        public ManufacturerDto Manufacturer { get; set; }
        // public List<SpecificitiesDto> Specificities { get; set; }
        public SpecificitiesDto Specificities { get; set; }
        public List<ProductDto> Products { get; set; }

    }
}

File: Views/src/components/stock-product/product/dtos/model-dto.ts
import { RootBase } from "src/shared/entities-dtos/root-base";
import { ManufacturerDto } from "./manufacturer-dto";
import { ProductDto } from "./product-dto";
import { SpecificitiesDto } from "./specificities-dto";

export class ModelDto extends RootBase {
    name: string;
    manufacturerId: number;
    manufacturer: ManufacturerDto;
    products: ProductDto[];
    specificitiesId: number;
    specificities: SpecificitiesDto;
}